src/iter/mod.rs

```diff
@@ -1352,6 +1354,25 @@ pub trait IndexedParallelIterator: ParallelIterator {
         interleave_shortest::new(self, other.into_par_iter())
     }
 
+    /// Split an iterator up into fixed-size chunks.
+    ///
+    /// Returns an iterator that returns `Vec`s of the given number of elements.
+    /// If the number of elements in the iterator is not divisible by `chunk_size`,
+    /// the last chunk may be shorter than `chunk_size`.
+    ///
+    /// Example:
```

Comment:
For consistency, let's make this `# Examples` - a header, and plural even though it's just one.